server/database/migrations/create_employees_table.js

```diff
@@ -12,7 +12,7 @@ class CreateEmployeesTable {
       + 'firstName VARCHAR (60) NOT NULL,'
       + 'lastName VARCHAR (60) NOT NULL,'
       + 'email VARCHAR (60) NOT NULL,'
-      + 'password VARCHAR (60) NOT NULL,'
+      + 'password VARCHAR (255) NOT NULL,'
```

Comment:
was there an issue with keeping the password at (60)

Reply:
Yes, because password encryption 
bcrypt generate a string with more than 60.

Reply:
Alright makes sense